Integrations/integration-Carbon_Black_Enterprise_Live_Response_CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+Added command ***cb-memdump*** replacing ***cb-memdeump***.
```

Comment:
From what I've seen you've also changed the memdump code, I believe this better be mentioned here - along with a description of the fixes you've made

Reply:
I'll add a description but I believe it should be relevant to cb-memdump as it is simply the same command but the name is fixed from the typo.